### PR TITLE
NO-ISSUE: dep ignore

### PR DESCRIPTION
**Pull request by eifrach**

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -17,6 +17,9 @@ updates:
       python-dependencies:
         patterns:
           - "*"
+        exclude-patterns:
+          - "opensearch-py"
+          - "kubernetes"
```
Comment (adriengentil) [inline .github/dependabot.yml:22]: can't we just tell dependabot to update the patch versions?

Comment (eifrach) [reply .github/dependabot.yml:22]: you can do something like this - but i need to test the it work's with groups
```yaml
    ignore:
      - dependency-name: "opensearch-py"
        versions: "18.*"
```

Comment (adriengentil) [reply .github/dependabot.yml:22]: could be worth to try? Up to you if you want to try in this PR or in follow PR.

Comment (eifrach) [reply .github/dependabot.yml:22]: k so let close this  for now i will work on some change

**Pull request closed without merging**